ui/src/app/pages/login/login.tsx

```diff
@@ -7,6 +7,9 @@ import { Header, SmallHeader } from 'app/components/headers';
 import { AouTitle } from 'app/components/text-wrappers';
 import colors from 'app/styles/colors';
 import { reactStyles } from 'app/utils';
+import { serverConfigStore } from 'app/utils/stores';
+
+import { LOGIN_ERROR_BANNER } from './login-error-banner';
```

Comment:
relative path

Reply:
This should be fine as both the files are in the same directory. Other egs controlled-tier-card

Reply:
relative is fine if it's logically related, like this is